calico_containers/pycalico/util.py

```diff
@@ -150,3 +154,61 @@ def validate_icmp_type(icmp_type):
     except ValueError:
         valid = False
     return valid
+
+def validate_hostname_port(hostname_port):
+    """
+    Validate the hostname and port format.  (<HOSTNAME>:<PORT>)
+    An IPv4 address is a valid hostname.
+
+    :return: Boolean: True if valid, False if invalid
+    """
+    # Should contain a single ":" separating hostname and port
+    if not isinstance(hostname_port, str):
+        _log.error("Must provide string for hostname:port validation, not: %s" % type(hostname_port))
+        return False
+    try:
```

Comment:
Aesthetically, I'd agree that it looks better, and I'll change it. I'm curious if there's a pep8 that also says there should be a new line here? And what's the gist of it? Always add a new line after conclusion of an if statement?